app/config/services.yml

```diff
@@ -1,83 +1,115 @@
-        tags:
-            - { name: kernel.event_listener, event: comment.created, method: onCommentCreated }
+        # The following settings will be appended to any local service definitions
+        # whose class matches this type. In this example, it will only apply to
+        # the "AppBundle\Controller\Admin\BlogController" service discovered by
+        # the "AppBundle\Controller\" rule below.
+        AppBundle\Controller\Admin\BlogController:
+            getters:
+                getSlugger: '@AppBundle\Utils\Slugger'
 
-    # Event subscribers are similar to event listeners but they don't need service tags.
-    # Instead, the PHP class of the event subscriber includes a method that returns
-    # the list of events listened by that class.
-    # See http://symfony.com/doc/current/event_dispatcher.html#creating-an-event-subscriber
-    app.requirements_subscriber:
-        class: AppBundle\EventListener\CheckRequirementsSubscriber
-        arguments: ['@doctrine.orm.entity_manager']
-        tags:
-            - { name: kernel.event_subscriber }
+    # This section enables to automatically register all classes found in the matching
+    # file paths and directories as services in the container. File and directory
+    # matching uses any valid glob pattern to create a white list of paths.
+    #
+    # In this example, the classes found in the following directories will be
+    # automatically registered as services:
+    #
+    # * src/AppBundle/EventListener/
+    # * src/AppBundle/Form/Type/
+    # * src/AppBundle/Security/
+    # * src/AppBundle/Twig/
+    # * src/AppBundle/Utils/
+    #
+    # The "resource" attribute can accept more specific glob patterns like
+    # "{EventListener/*Subscriber.php,Form/Type/*Type.php}".
+    AppBundle\:
+        # Register all classes in the src/AppBundle directory as services
+        resource: '../../src/AppBundle/{EventListener,Form/Type,Security,Twig,Utils}'
 
-    # To inject the voter into the security layer, you must declare it as a service and tag it with security.voter.
-    # See http://symfony.com/doc/current/security/voters.html#configuring-the-voter
-    app.post_voter:
-        class: AppBundle\Security\PostVoter
-        public: false
-        tags:
-            - { name: security.voter }
+    # The other section defines a rule to automatically register and autowire the
+    # controller classes found in the src/AppBundle/Controller/ directory.
+    #
+    # By default all services are made private according to the global "_defaults"
+    # section at the top of this file.
+    #
+    # However, in Symfony, controllers must always be declared public in order to
+    # be lazy instanciated when they're really needed. This is why the inherited
+    # default "public" attribute is overriden to force the registered controller
+    # services to be marked public.
+    AppBundle\Controller\:
```

Comment:
I'm sorry but we are not going to get rid of AppBundle in the Symfony Demo. For this kind of changes, the Symfony Demo must go behind the Symfony Framework itself. If Symfony decides to go bundle-less for Symfony 4.0 (I wish it will) then we'll update the Symfony Demo accordingly. But we can't do that before. I'm sorry.